src/Functions/multiplyDecimal.cpp

```diff
@@ -69,21 +69,25 @@ In case you don't really need controlled precision and/or need fast computation,
 :::
     )";
     FunctionDocumentation::Syntax syntax = "multiplyDecimal(a, b[, result_scale])";
-    FunctionDocumentation::Argument argument1 = {"a", "First value. Type [Decimal](/sql-reference/data-types/decimal)."};
-    FunctionDocumentation::Argument argument2 = {"b", "Second value. Type [Decimal](/sql-reference/data-types/decimal)."};
-    FunctionDocumentation::Argument argument3 = {"result_scale", "Scale of result. Type [Int/UInt](/sql-reference/data-types/int-uint)."};
-    FunctionDocumentation::Arguments arguments = {argument1, argument2, argument3};
-    FunctionDocumentation::ReturnedValue returned_value = "The result of multiplication with the given scale. Type: [Decimal256](/sql-reference/data-types/decimal).";
-    FunctionDocumentation::Example example1 = {"Usage example", "SELECT multiplyDecimal(toDecimal256(-12, 0), toDecimal32(-2.1, 1), 1)", "25.2"};
-    FunctionDocumentation::Example example2 = {"Difference with regular multiplication", "SELECT multiplyDecimal(toDecimal256(-12, 0), toDecimal32(-2.1, 1), 1)", R"(
+    FunctionDocumentation::Arguments arguments =
```

Comment:
I'll not be a code formatting n*** (i.e. it is okay to submit the PR w/o reformatting) but the "correct" way to format this would be

```cpp
    FunctionDocumentation::Arguments arguments = {
        {"a", "First value.", {"Decimal"}},
        {"b", "Second value.", {"Decimal"}},
        {"result_scale", "Scale of result.", {"Int*", "UInt*"}}
    };
```

(just keep in mind for future PRs)

Reply:
Okay, I wasn't sure because in some cases the style check seems to complain without `{` on a new line.